field_manager/fm_util.F90

```diff
@@ -102,17 +94,6 @@ module fm_util_mod  !{
 !        Interface definitions for overloaded routines
 !
 
-!interface  fm_util_get_value  !{
```

Comment:
This commented out interface declaration was removed because the procedures under it are not present in the module.